changelog.d/2014.added.md

```diff
@@ -0,0 +1 @@
+Add k0s containerd socket path.
```

Comment:
I'd change the wording a bit -
Added k0s support - add k0s containerd socket path.